docs/concepts/README.md

```diff
@@ -176,7 +176,7 @@ You can also build your own language packs to customize the output container to
 ### Function Deployment and Lifecycle
 
 <!-- TODO: use img: shapes for these icons -->
-```mermaid
```

Comment:
I'm surprised we need this space - this isn't usually the norm in our code snippets?

Reply:
I'm not sure, there was a second syntax issue, but our other use of mermaid did the same thing.